src/components/Card/card.css

```diff
@@ -35,8 +35,8 @@
     }
 }
 .lil-stat-card {
-    box-shadow: 8px 8px 60px 4px rgba(45, 45, 45, 0.08);
-    @apply lil-p-5 lil-bg-white lil-rounded-xl lil-h-auto;
+    width: 95%;
```

Comment:
plz dont use random values for it...let it be 100% and parent will take care how to adjust the size